corehq/ex-submodules/casexml/apps/case/cleanup.py

```diff
@@ -62,7 +59,6 @@ def rebuild_case_from_forms(domain, case_id, detail):
     return FormProcessorInterface(domain).hard_rebuild_case(case_id, detail)
 
 
-@unit_testing_only
```

Comment:
The fact that there are unit-testing-only functions mixed with production code is a bit disconcerting.